source/MapPanel.cpp

```diff
@@ -996,8 +1012,9 @@ void MapPanel::DrawTravelPlan()
 
 		Point from = Zoom() * (next->Position() + center);
 		Point to = Zoom() * (previous->Position() + center);
-		Point unit = (from - to).Unit() * LINK_OFFSET;
-		LineShader::Draw(from - unit, to + unit, 3.f, drawColor);
+		float linkOffset = (isStarry == true) ? LINK_OFFSET : 0;
```

Comment:
Replace the 0 by 0.f, which will mark it as a float. Otherwise it will be converted from int (which is what this 0 is) to float
Alsp no need to store it in a variable since you dont ise it again afterwards. You can do this next line.
Its okay to name variables for clarity, but here its the same name anyway